AppCenterReactNativeShared/android/src/main/java/com/microsoft/appcenter/reactnative/shared/AppCenterReactNativeShared.java

```diff
@@ -58,6 +59,17 @@ public static synchronized void configureAppCenter(Application application) {
             AppCenterLog.debug(LOG_TAG, "Configure with secret.");
             AppCenter.configure(application, sAppSecret);
         }
+
+        /*
+         * In case when sStartAutomatically flag is set, we are sure that Auth service will be started
+         * with other services, but by separately `start` call. So, we don't need to reset the auth token
+         * after first start call.
+         */
+        try {
+            Class.forName("com.microsoft.appcenter.auth.Auth");
+            AuthTokenContext.getInstance().doNotResetAuthAfterStart();
+        } catch (Exception ignored) {
+        }
```

Comment:
Add a comment (example [here](https://github.com/microsoft/appcenter-sdk-android/blob/develop/apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java#L375)).

`/* Nothing to handle; this is reached if Auth SDK isn't used. */`